Changelog.md

```diff
@@ -1,10 +1,15 @@
-[0.11.0.0] —
- * [Add `indexMaybe` and synonym `(!?)` for indexing that returns `Maybe`](https://github.com/haskell/bytestring/pull/261)
- * Add `unsafePackLiteral` to `Data.ByteString.Internal`. When possible, use
+[0.11.0.0] — September 2020
+ * [Change internal representation of `ByteString`, removing offset](https://github.com/haskell/bytestring/pull/175)
```

Comment:
It might be good to add a sentence about the `PS` pattern synonym here. E.g.

```suggestion
 * [Change internal representation of `ByteString`, removing offset](https://github.com/haskell/bytestring/pull/175)
   * The old `PS` constructor has been turned into a pattern synonym that is available with GHC >= 8.0 for backwards compatibility.
```